thread/test/perf_workpool.cpp

```diff
@@ -30,9 +30,22 @@ DEFINE_uint64(fires, 80000, "How many tasks to fire");
 static photon::WorkPool* pool;
 static std::atomic<uint64_t> sum_time;
 
-void task_async() {
+__attribute__ ((noinline))
+static void do_fill(char* buf, size_t size) {
+    memset(buf, 0, size);
+}
+// A function that costs approximately 1 us
+static void func_1us() {
+    for (int i = 0; i < 5; ++i) {
+        char buf[32 * 1024];
+        do_fill(buf, sizeof(buf));
```

Comment:
It's better to write a loop and repeatedly query time.

Reply:
ok